Examples/4-orm/GreetingsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrmService.Entities;
+using ViennaNET.Orm.Application;
+using ViennaNET.Utils;
+
+namespace EmptyService
```

Comment:
Empty service
you may just use reshaper/vs command which will fix namespace issues in whole project

Reply:
Fixed